sdk/redisenterprise/Azure.ResourceManager.RedisEnterprise/CHANGELOG.md

```diff
@@ -1,12 +1,12 @@
 # Release History
 
-## 1.2.0-beta.1 (Unreleased)
+## 1.2.0-beta.1 (2024-09-01-preview)
```

Comment:
the content in the paratheses is the target release date for the SDK, see: https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=signing-and-publishing-nuget-packges for details.